vagrant/horizontal-app-scaling/jobs/traefik.nomad

```diff
@@ -0,0 +1,62 @@
+job "traefik" {
+  datacenters = ["dc1"]
+
+  group "traefik" {
+    count = 1
+
+    network {
+      port  "admin"{
+        static = 8081
+      }
+      port  "autoscaler"{
+        static = 8080
+      }
+      port "grafana" {
+        static = 3000
+      }
+      port "prometheus" {
+        static = 9090
+      }
+      port  "webapp"{
+         static = 8000
+      }
+    }
+
+    service {
+      name = "traefik-http"
+      provider = "nomad"
+      port = "webapp"
+    }
+
+    service {
+      name = "traefik-admin"
+      provider = "nomad"
+      port = "admin"
+      tags = [
+        "metrics"
+      ]
+    }
+
+    task "server" {
+      driver = "docker"
+      config {
```

Comment:
I think it would be good to keep `network_mode = "host` here so that we can access Traefik using `127.0.0.1` as currently [defined in the instructions](https://learn.hashicorp.com/tutorials/nomad/autoscaler-vagrant-demo?in=nomad/autoscaler#generate-load-and-scale-up).

Reply:
sure i am willing to wait for the checks to be part of nomad native sd. I ll see how far i get with nomad 1.3.5. Will fix this one, don’t think i did this deliberately, prob just a copy paste error.